spec/features/homepage_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "rails_helper"
+
+feature "homepage" do
+  scenario "enables me to view 'smartflix'" do
```

Comment:
Typically a scenario would be a description of what the user is doing, so something like "A user visits the homepage"

Reply:
Thank you for your suggestion! I changed the scenario accordingly.